.changeset/fast-ads-tease.md

```diff
@@ -0,0 +1,7 @@
+---
+'sku': patch
+---
+
+Enable `babel-loader` cache
+
+Sku's webpack configuration now configues `babel-loader` to emit a cache to `node_modules/.cache/babel-loader`. The primary use case for this cache is speeding up production builds. It can also speed up local development in situations where the webpack cache is invalidated.
```

Comment:
```suggestion
Sku's webpack configuration now configures `babel-loader` to emit a cache to `node_modules/.cache/babel-loader`. The primary use case for this cache is speeding up production builds. It can also speed up local development in situations where the webpack cache is invalidated.
```